__tests__/endpoint.tests.js

```diff
@@ -99,8 +99,40 @@ describe('GET/api/articles', () => {
         const expectedColumns = ['article_id', 'title', 'topic', 'author', 'created_at', 'votes', 'article_img_url', 'comment_count'];
 
         expect(result.status).toBe(200);
-        expect(result.body).toHaveLength(13);
-        expect(result.body).toBeSortedBy('created_at', {descending:true});
-        expect(Object.keys(result.body[0])).toEqual(expectedColumns);
+        expect(result.body.articles).toHaveLength(13);
+        expect(result.body.articles).toBeSortedBy('created_at', {descending:true});
+        expect(Object.keys(result.body.articles[0])).toEqual(expectedColumns);
     })
 })
+
+describe('GET/api/articles/:article_id/comments', () => {
+    it('200: responds with an array of comments with appropriate columns, sorted by created_at in descending order', async () => {
+        const result = await request(app).get('/api/articles/1/comments');
+        const expectedColumns = ['comment_id', 'body', 'article_id', 'author', 'votes', 'created_at'];
+        expect(result.status).toBe(200);
+        expect(result.body.comments).toHaveLength(11);
+        expect(result.body.comments).toBeSortedBy('created_at', {descending:true});
+        expect(Object.keys(result.body.comments[0]).sort()).toEqual(expectedColumns.sort());
+    })
```

Comment:
Nice 👍 You may also want to check the values of the keys too e.g. that author is a string and votes is a number etc.

Reply:
You will also need to check all of the comments, not just the first. A loop may be needed here :)